test/newton/newton_tests.jl

```diff
@@ -0,0 +1,17 @@
+using OrdinaryDiffEq
+using DiffEqDevTools
+using DiffEqBase
+using LineSearches
+using Test
+
+import ODEProblemLibrary: prob_ode_orego, prob_ode_rober, prob_ode_lorenz
```

Comment:
This isn't enough for a Newton subgroup. Add it to the interface subgroup (needs to be in runtests.jl).